### DISCUSSION TITLE
Step conditional on status transition from failure() to success()?

### DISCUSSION THREAD

**Discussion by britzl** [community/community#25050] [Actions]
<p>We send a notification on Slack when a job fails. This is detected using an if: failure() conditional check. Now I’d also like to send a Slack notification when the job transitions from failure() to success() but not on every successful build. Is this kind of state transition possible to detect?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi britzl, </p>
<p>Sorry to tell you that there is not a event to track job status transition. So your requirement could not be reached currently.</p>

**Comment by kornholi** [comment]
<p>Any plans for such a feature? This is the only annoyance we have ran into while transitioning away from Travis. A verified/endorsed action for Slack notifications would be nice as well.</p>

**Comment by Yanjingzhu** [comment]
<p>Hi kornholi,</p>
<p>If you have feedback or feature requests for GitHub Actions, share those in the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">Feedback form for GitHub Actions</a>.</p>
